.github/workflows/comment.yaml

```diff
@@ -60,9 +61,10 @@ jobs:
       - name: Download artifacts
         uses: actions/download-artifact@v4.1.8
         with:
+          pattern: "${{ env.ARTIFACT_NAME }}-*"
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The GH token is required for `workflow-runs` according to [download-artifacts](https://github.com/actions/download-artifact/tree/v4.1.8?tab=readme-ov-file#inputs) inputs